recipes/dynare_preprocessor/run_test.sh

```diff
@@ -0,0 +1,4 @@
+set -ex
+
+dynare-preprocessor | grep -q "Missing model file!"
+dynare-preprocessor example1.mod | grep -q "Preprocessing completed."
```

Comment:
Nothing is being grepped. Doesn't that mean your test is failing?

Reply:
No. When you run the preprocessor without modem arguments, it should print that model file is missing. When the right kind of model file is supplied the output message contains "preprocessing completed". In both cases it returns an error if output is not what I'd expected.